cooltools/api/rearrange.py

```diff
@@ -92,7 +95,7 @@ def _flip_bins(regdf):
     return regdf
 
 
-def _reorder_bins(
+def _rearrange_bins(
```

Comment:
this is useful/general enough to drop the underscore and add a docstring

Reply:
would this function be shorter if one were to use a lookup table to convert old bin_ids-> new bin_ids? Such a lookup table can be easily generated in rearrange_bins